pyproject.toml

```diff
@@ -16,16 +17,18 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "dask",
+    "dask[dataframe]",
+    "dask[distributed]",
```

Comment:
I believe that `dask[distributed]` is not a strict requirement, but something we'd like user to have to be able to run things in parallel with Dask client. User may still run LSDB without Dask client, 1) with no client and not in parallel, 2) with [Dask on Ray](https://docs.ray.io/en/latest/ray-more-libs/dask-on-ray.html), 3) maybe something else.

I would make a comment here about it.